props/saber_fett263_buttons.h

```diff
@@ -1797,6 +1797,7 @@ SaberFett263Buttons() : PropBase() {}
 
   void Setup() override {
     RestoreGestureState();
+    sound_library_.init();
```

Comment:
The idea is to call init on the highest version of the sound library you actually use, so this should probably be:

```
#ifdef MOUNT_SD_SETTING
   sound_library_v2.init();
#else
   sound_library_.init();
#endif
```

Reply:
Updated.